packages/location-field/src/index.tsx

```diff
@@ -784,73 +868,71 @@ const LocationField = ({
   // or if the input field has text.
   const clearButton =
     showClearButton && location ? (
-      <S.InputGroupAddon>
-        <S.Button
-          aria-label={intl.formatMessage({
-            id: "otpUi.LocationField.clearLocation"
-          })}
-          onClick={onClearButtonClick}
-        >
-          {clearButtonIcon}
-        </S.Button>
-      </S.InputGroupAddon>
+      <S.ClearButton
+        aria-label={intl.formatMessage({
+          id: "otpUi.LocationField.clearLocation"
+        })}
+        onClick={onClearButtonClick}
+      >
+        {clearButtonIcon}
+      </S.ClearButton>
     ) : null;
-  if (isStatic) {
-    // 'static' mode (menu is displayed alongside input, e.g., for mobile view)
-    return (
-      <div className={className}>
-        <S.FormGroup>
-          <S.InputGroup role="group">
-            <S.InputGroupAddon>
-              <LocationIconComponent locationType={locationType} />
-            </S.InputGroupAddon>
-            {textControl}
-            {clearButton}
-          </S.InputGroup>
-        </S.FormGroup>
-        <S.HiddenContent role="status">{statusMessages?.join(", ")}</S.HiddenContent>
-        <S.StaticMenuItemList
-          // Hide the listbox from assistive technology if no valid items are shown.
-          aria-hidden={menuItemCount === 0 || undefined}
+
+  const ItemList = isStatic ? S.StaticMenuItemList : S.MenuItemList;
+
+  return (
+    <S.InputGroup className={className} onBlur={onBlurFormGroup} role="group">
+      <S.DropdownButton
+        aria-controls={controlledId}
+        aria-expanded={shouldRenderList}
+        aria-label={intl.formatMessage({
+          defaultMessage: "Open the list of location suggestions",
+          description:
+            "Text to show as a a11y label for the button that opens the dropdown list of locations",
+          id: "otpUi.LocationField.suggestedLocationsLong"
+        })}
+        onClick={onDropdownToggle}
+        tabIndex={-1}
+      >
+        <LocationIconComponent locationType={locationType} />
+      </S.DropdownButton>
+      {textControl}
+      {clearButton}
+      {/* Note: always render this status tag regardless of the open state,
+          so that assistive technologies correctly set up status monitoring. */}
+      <S.HiddenContent role="status">
+        {statusMessages.join(", ")}
```

Comment:
Can we do this across all languages? I'm not too fussed since it's a very rarely seen feature.

Reply:
Addressed in https://github.com/opentripplanner/otp-ui/pull/533/commits/6913f20a27f2942c71d290c89196c15fb9a5eaba, although I am not thrilled with what that does in Spanish.